lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -1124,6 +1124,16 @@ export class StandardController implements IController {
         }
     }
 
+    /**
+     * Creates a cache interaction client to clear broswer cache.
+     * @param logoutRequest
+     */
+    async clearCache(logoutRequest?: EndSessionRequest): Promise<void> {
+        const correlationId = this.getRequestCorrelationId(logoutRequest);
```

Comment:
if `logoutRequest` is optional, all of these are undefined. What is the default behavior?

Reply:
This API exposes clearCacheOnLogout() which clears only account related items from cache if account is provided in the logout request. And if account or the logout request is not provided, it will clear out the entire cache by default.

Reply:
getRequestCorrelationId will either return the correlationId passed in or create a new one if it was undefined